aikau/src/main/resources/alfresco/services/SiteService.js

```diff
@@ -30,15 +30,16 @@ define(["dojo/_base/declare",
         "alfresco/core/NotificationUtils",
         "alfresco/core/ObjectTypeUtils",
         "alfresco/core/topics",
+        "alfresco/core/WidgetsOverrideMixin",
```

Comment:
Great idea with the mixin. Have you got tickets to add this mixin to other widgets that include default widget config?